.evergreen/config.yml

```diff
@@ -1045,7 +1027,10 @@ functions:
       params:
         script: |
           # Remove all Docker images
-          docker rmi -f $(docker images -a -q) &> /dev/null || true
+          DOCKER=$(command -v docker) || true
```

Comment:
Trying to get an understanding of `command -v docker`. Does this check if the `docker` `PATH` has been defined or that the command exists in the shell?

Overall, a cool `bash` technique I've just learned! #TIL

Reply:
It searches for an executable of that name on your PATH.